.github/project.yml

```diff
@@ -1,3 +1,3 @@
 release:
-  current-version: 7.0.0-alpha2
+  current-version: 7.0.0-alpha5
```

Comment:
```suggestion
  current-version: 7.0.0-alpha3
```

We don't have 3 and 4 🤔

Reply:
yes, but the one in the schema is the alpha5, I think is better to align with it, isnt it?